docs/reference/glossary/network-latency-management.md

```diff
@@ -0,0 +1,112 @@
+---
+id: network-latency-management
+title: Network Latency Management
+sidebar_label: Network Latency Management
+---
+
+Every player is affected by a certain amount of latency, which is the result of the distance to the server/host, the number of hops a packet has to make, and the server's tick rate and update rate.
+
+This is where Network Latency Management comes in. Due to these delays, you would see where a player was a few milliseconds ago, not where they are right now. If the game didn't compensate for at least a portion of this delay, then you would be unable to hit other players.
+
+
+
+## Server rewind
+
+The server uses the latency of the player (including any inherent delay due to interpolation) to rewind time by an appropriate amount in order to determine what the shooting client saw at the time the shot was fired.
```

Comment:
Lag compensation does not use latency. It just does rewinds time appropriately. The underlying algorithm functions in a different way which I cannot be explain in a short paragraph.

Reply:
text updated to remove latency references